### DISCUSSION TITLE
Reset energy meter Xiaomi SP-EUC01

### DISCUSSION THREAD

**Discussion by flaektrem** [Koenkk/zigbee2mqtt#8832] [Devices]
Hi there,

Anyone know if this is possible? I've searched the web without answer

**Comment by k8ieone** [comment]
I'm also looking for an answer to this. Normally with Tuya smart plugs it's enough to send `{"reset":""}` to `zigbee2mqtt/DEVICE NAME/set`. I've tried this with the Aqara plug and it doesn't do anything.

The only way that worked was to reset the plug (hold the power button for 5 seconds), but that causes the plug to completely reset and leave the network, which is really inconvenient.
